irony-cdb-json.el

```diff
@@ -100,6 +111,30 @@ it could easily be used with helm, for instance, by enabling
     (irony-cdb-json--save-project-alist)
     (irony-cdb-autosetup-compile-options)))
 
+(defun irony-cdb-json--last-mod (file)
+  (nth 5 (file-attributes file)))
```

Comment:
You could simplify the handling of non-existing file by returning a null time value when the file does not exists:

```el
(or (nth 5 (file-attributes file))
      '(0 0 0 0))
```